main.go

```diff
@@ -144,7 +260,9 @@ func main() {
 		*debug = false
 		tmpH = nil
 	}
-	//bot.Debug = true
+    if !(*debug) {
```

Comment:
now i run fmtgo -w in last commit before send you a fixed pull request.
i dident know of it existence before you talking here